futures-util/src/stream/pending.rs

```diff
@@ -0,0 +1,29 @@
+use core::marker;
```

Comment:
This module does not seem to be defined...

Reply:
:man_facepalming: Fixed...